packages/runtime-core/src/apiWatch.ts

```diff
@@ -253,12 +255,16 @@ function doWatch(
         if (cleanup) {
           cleanup()
         }
```

Comment:
This also needs to be wrapped by `pause/reset tracking`

Reply:
Done and thanks for reminding.